src/components/SecondaryPanes/XHRBreakpoints.css

```diff
@@ -7,12 +7,12 @@
 }
 
 .xhr-input {
-  width: 100%;
+  width: 70%;
```

Comment:
maybe flexbox would be a good fit here?

Reply:
Good idea; we can use flexbox to take up available space after before the `<select>` element.

Reply:
Sounds good - changed the form to `display: flex`, and the the URL input box now takes up all the space before `<select>`